### PR TITLE
Feat/lint without args

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
This PR adds ability to `lint` command to load local files from super.json

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have updated the documentation accordingly. For updating Oclif commands documentation use [oclif-dev](https://github.com/oclif/dev-cli#oclif-dev-readme).
- [x] I have read the **CONTRIBUTION_GUIDE** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/commands/lint.ts [hunk 1]
```diff
@@ -86,13 +86,31 @@ export default class Lint extends Command {
...
 
-    if (flags.output !== '-' && flags.output !== '-2') {
-      //Warn user
-      this.log(
-        yellow(
-          'You are using a hidden command. This command is not intended for public consumption yet. It might be broken, hard to use or simply redundant. Tread with care.'
-        )
+    let files: string[] = [];
+    if (!argv || argv.length === 0) {
+      const superPath = await detectSuperJson(process.cwd());
```
Comment (martinalbert) [inline src/commands/lint.ts:91]: Wondering whether author would use this command in more nested folders, don't we want to add `--scan` flag as `install` command has?

File: src/commands/lint.ts [hunk 2]
```diff
@@ -86,13 +86,31 @@ export default class Lint extends Command {
...
+      //Load super json
+      const loadedResult = await SuperJson.load(joinPath(superPath, META_FILE));
+      const superJson = loadedResult.match(
+        v => v,
+        err => {
+          throw userError(`Unable to load super.json: ${err}`, 1);
+        }
       );
+      for (const profile of Object.values(superJson.normalized.profiles)) {
+        if ('file' in profile) {
```
Comment (martinalbert) [inline src/commands/lint.ts:104]: Don't we also want to check whether grid folder does not contain profile?

**Review by martinalbert — APPROVED** [review]
Command still has quiet and help flag defined and don't have examples, but other than that, looks good. 👍🏻

File: src/commands/lint.ts (hunk 1 above)
Comment (Jakub-Vacek) [reply src/commands/lint.ts:91]: Yeah, that's good point.

File: src/commands/lint.ts [hunk 3]
```diff
@@ -86,13 +86,31 @@ export default class Lint extends Command {
...
-      //Warn user
-      this.log(
-        yellow(
-          'You are using a hidden command. This command is not intended for public consumption yet. It might be broken, hard to use or simply redundant. Tread with care.'
-        )
+    let files: string[] = [];
+    if (!argv || argv.length === 0) {
+      const superPath = await detectSuperJson(process.cwd());
+      if (!superPath) {
+        throw userError('Unable to compile, super.json not found', 1);
```
Comment (TheEdward162) [inline src/commands/lint.ts:93]: Compile?

**Review by martinalbert — APPROVED** [review]

**Pull request merged**